Add Anitya external version checker

xylo04 opened the pull request:
This should let https://github.com/flathub/flatpak-external-data-checker automatically file issues when the upstream version is updated. This tracks fldigi via https://release-monitoring.org/project/818/

xylo04 committed:
Add Anitya external version checker

This should let https://github.com/flathub/flatpak-external-data-checker automatically file issues when the upstream version is updated. This tracks fldigi via https://release-monitoring.org/project/818/

AsciiWolf commented on com.w1hkj.fldigi.yml:
```diff
@@ -41,6 +41,10 @@ modules:
       - type: archive
         url: http://www.w1hkj.com/files/fldigi/fldigi-4.1.16.tar.gz
```
```suggestion
        url: http://www.w1hkj.com/files/fldigi/fldigi-4.1.18.tar.gz
```

AsciiWolf commented on com.w1hkj.fldigi.yml:
```diff
@@ -41,6 +41,10 @@ modules:
       - type: archive
         url: http://www.w1hkj.com/files/fldigi/fldigi-4.1.16.tar.gz
         sha256: 640e3ea544adc7696e18cb08ed311ae1971ed18c609c39b55fb3c40c34dcd3b5
```
```suggestion
        sha256: 3c7fd84ab3a84ba4525b251e9d56120daee626a3831684a974fdad476f90270d
```

AsciiWolf reviewed (changes requested):
Thanks for your PR! This should fix the build.

xylo04 committed:
Update fldigi version

The old version can no longer be built (at least, not via the tar.gz) because it's been removed from w1hkj's website.

Co-authored-by: AsciiWolf <mail@asciiwolf.com>
